patches/extra/ungoogled-chromium/add-extra-channel-info.patch

```diff
@@ -0,0 +1,30 @@
+description: change extra channel info to "Ungoogled Chromium"
+author: Wengling Chen <feiyu2817@gmail.com>
+
+---
+ chrome/common/channel_info_posix.cc | 2 +-
+ chrome/common/channel_info_win.cc   | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+--- a/chrome/common/channel_info_posix.cc
++++ b/chrome/common/channel_info_posix.cc
+@@ -21,7 +21,7 @@ version_info::Channel GetChannelImpl(std::string* modifier_out) {
+   version_info::Channel channel = version_info::Channel::UNKNOWN;
+   std::string modifier;
+ 
+-  char* env = getenv("CHROME_VERSION_EXTRA");
++  char* env = "Ungoogled Chromium";
```

Comment:
This would prevent reading the environment variable `CHROME_VERSION_EXTRA`.

Reply:
This is what Fedora uses in their `chromium` package so I just adapted it. Is there any actual a use case for `CHROME_VERSION_EXTRA`?

Reply:
> This is what Fedora uses in their chromium package so I just adapted it. Is there any actual a use case for CHROME_VERSION_EXTRA? Although appending shall be possible since the extra info is just a string.

Debian also sets it to indicate the build distribution & running distribution. I don't want to break that behavior.

Actually, could we just set `(ungoogled-chromium)` only if no extra channel info is provided at all? I don't want to force users to have this string if they don't want it.